### DISCUSSION TITLE
Avoid translating code regions

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [mantinedev/mantine#1872] [Feature requests]
## Suggest

I'm a developer in a non-English speaking country and I'm trying to learn mantine. Mantine's website is very nice, but it has one downside.

If you use Google Translate, it translates well, but it is inconvenient because it translates even the code area.

No other sites have had this issue.

## Problems that arise

![image](https://user-images.githubusercontent.com/27342882/180989850-8f25ce3a-3c56-4b8e-8b67-f39dce63b410.png)

## Wish

I hope the code area is not translated.

## Etc

I know that there are many English-speaking users, so this may be an unnecessary request, but I'm requesting the feature in hopes that more users will come in.

Cheers to the mantine development group.

## Note that

Code regions are not translated in edge browser, only that problem occurs in chrome translator.

**Comment by rtivital** [comment]
I'll add `translate="no"` attribute to the Prism component in the next patch.

**Reply by ryuhangyeong** [reply]
Thanks for your kind reply.

If given the opportunity, may I create a PR that addresses the issue?

when i looked

It seems to be possible to solve it by simply doing `<Box translate="no" />` in src/mantine-prism/src/Prism .

May I try?

**Reply by rtivital** [reply]
Yes, sure
